src/Parser/Hunter/Marksmanship/Modules/Talents/CallingTheShots.js

```diff
@@ -34,6 +34,7 @@ class CallingTheShots extends Analyzer {
       if (this.spellUsable.cooldownRemaining(SPELLS.TRUESHOT.id) < COOLDOWN_REDUCTION_MS) {
         const effectiveReductionMs = this.spellUsable.cooldownRemaining(SPELLS.TRUESHOT.id);
         this.effectiveTrueshotReductionMs += effectiveReductionMs;
+        this.spellUsable.reduceCooldown(SPELLS.TRUESHOT.id, effectiveReductionMs);
```

Comment:
`reduceCooldown` returns the effective cd reduction it applied, so you could use it instead of the cooldownRemaining statement on line 35